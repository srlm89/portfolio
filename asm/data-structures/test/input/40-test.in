ERASE actual/40-test.out
INSERT 3 2.988470 Basic Instinct 720p
PRINT_IN_ORDER actual/40-test.out
INSERT 4 6.096495 Full.Metal.Jacket.REMASTERED.1987.720p.HDDVD.x264-SEPTiC
INSERT 24 9.143032 Miami Vice 720p
REMOVE_ID 24
PRINT_IN_ORDER actual/40-test.out
INSERT 30 4.867845 Shutter   720p
INSERT 0 6.216898 007 The Man With the Golden Gun [1974] [BluRay] [720p] [DTS]
INSERT 14 6.392917 Blood River 720P
REMOVE_ID 30
PRINT_IN_ORDER actual/40-test.out
INSERT 36 1.030676 Pursuit_Of_Happyness[2006]   720p
INSERT 7 8.231199 A Bug's Life [720p]
INSERT 32 1.879655 Blade 2   720p
INSERT 22 8.530070 James Bond 08 - Live and Let Die [1973]
REMOVE_ID 22
REMOVE_ID 7
PRINT_IN_ORDER actual/40-test.out
INSERT 47 1.412558 Stargate-The Ark of Truth 720p
INSERT 2 5.955858 Hancock    1080p
INSERT 39 5.275730 Chicken Run [2000] [720p]
INSERT 26 5.966941 Elite Squad 1080p
INSERT 40 3.385730 Deep Sea
REMOVE_ID 40
REMOVE_ID 39
PRINT_IN_ORDER actual/40-test.out
INSERT 52 5.258948 Crash    1080p
INSERT 34 0.466349 SAW II   720p
INSERT 12 5.559937 Connected   720p
INSERT 46 1.135494 Rocky 3 [1982] [BluRay] [720p] [AC3]
INSERT 58 7.807691 Fast and the Furious 1 - 1080p
INSERT 48 5.298127 X-Men 3    720p
REMOVE_ID 32
REMOVE_ID 34
REMOVE_ID 0
PRINT_IN_ORDER actual/40-test.out
INSERT 6 1.928617 Minority Report [720p]
INSERT 11 4.479115 X-Men 1    720p
INSERT 61 3.196179 Blade 1   720p
INSERT 31 1.814512 Hitman    720p
INSERT 65 6.894192 James Bond 12 - For Your Eyes Only [1981]
INSERT 20 0.345042 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN]
INSERT 33 6.628069 The.Warlords.2007.720p.BluRay.DTS.x264-ESiR
REMOVE_ID 36
REMOVE_ID 20
REMOVE_ID 61
PRINT_IN_ORDER actual/40-test.out
INSERT 60 8.774373 Apollo 13   1080p
INSERT 42 9.555306 The Secret of Moonarce [720p]
INSERT 70 8.776040 Final Fantasy The Spirits Within    720p
INSERT 8 9.164452 Disturbia    720p
INSERT 62 4.644306 3_10 to Yuma   720p
INSERT 45 0.147915 SAW II   720p
INSERT 67 2.589482 Alvin And The Chipmunks [720p]
INSERT 15 9.549769 Pursuit_Of_Happyness[2006]   720p
REMOVE_ID 31
REMOVE_ID 6
REMOVE_ID 47
REMOVE_ID 8
PRINT_IN_ORDER actual/40-test.out
INSERT 71 5.903459 X-Men Origin - Wolverine [2009] [1080p]
INSERT 77 5.391743 Red Cliff    1080p
INSERT 44 0.140376 Home    1080p
INSERT 78 1.748962 Fast and the Furious 2 - 1080p
INSERT 51 1.485627 Kung Fu Panda Secrets of the Furious Five    720p
INSERT 23 7.382940 James Bond 17 - Golden Eye [1995]
INSERT 13 1.651002 Top.Gun.1986.720p.HDDVD.x264-SEPTiC
INSERT 53 2.400935 Finding Nemo [2003] [720p]
INSERT 68 9.530507 Disturbia    720p
REMOVE_ID 26
REMOVE_ID 70
REMOVE_ID 13
REMOVE_ID 4
PRINT_IN_ORDER actual/40-test.out
INSERT 85 0.584869 Kung Fu Panda Secrets of the Furious Five    720p
INSERT 35 4.908208 Orphan (2009)    1080p
INSERT 82 8.423432 Green Lantern First Flight   720p
INSERT 41 2.318450 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p
INSERT 50 6.673878 Breach 1080p
INSERT 54 5.593976 Behind Enemy Line 2001   720p
INSERT 56 9.233027 Oceans.Twelve.2004.720p.HDDVD.x264-CDDHD
INSERT 59 7.372526 Kingdom of Heaven [2005] [720p]
INSERT 21 3.593027 Defiance.1080p
INSERT 89 4.807683 Hellboy 1    1080p
REMOVE_ID 58
REMOVE_ID 14
REMOVE_ID 44
REMOVE_ID 78
REMOVE_ID 68
PRINT_IN_ORDER actual/40-test.out
INSERT 73 0.781529 Gladiator 1080p
INSERT 66 2.049403 Harry Potter Collection    720p
INSERT 57 1.935369 Face.Off.1997.720p
INSERT 88 3.300553 New York I Love You 720p   720p
INSERT 37 8.050939 Darko 720P
INSERT 92 7.082095 The Matrix Reloaded   1080p
INSERT 83 2.891269 Armageddon [1998] [720p]
INSERT 84 7.031197 Public Enemies [720p]
INSERT 94 4.844811 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p
INSERT 101 9.016673 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL
INSERT 64 1.827436 I Am Legend    720p
REMOVE_ID 77
REMOVE_ID 67
REMOVE_ID 94
REMOVE_ID 41
REMOVE_ID 66
PRINT_IN_ORDER actual/40-test.out
INSERT 28 2.529464 Bride Wars   720p
INSERT 72 4.877273 Hero    1080p
INSERT 80 4.827346 James Bond 20 - Die Another Day [2002] [720p]
INSERT 97 3.057022 The Mummy Tomb Of The Dragon Emperor   720p
INSERT 102 6.285936 Fantastic Four   1080p
INSERT 104 7.788449 Scooby.Doo.The.Mystery.Begins.2009   720p
INSERT 108 8.044182 Chicken Run [2000] [720p]
INSERT 113 8.562154 Around The World In 80 Days [2004] [720p]
INSERT 74 6.730348 Red Dragon    1080p
INSERT 105 8.374442 James Bond 01 - Dr. No [1962] [720p]
INSERT 100 9.400373 Swordfish    720p
INSERT 81 6.930153 James Bond 15 - The Living Daylights [1987] [720p]
REMOVE_ID 82
REMOVE_ID 81
REMOVE_ID 84
REMOVE_ID 11
REMOVE_ID 85
REMOVE_ID 3
PRINT_IN_ORDER actual/40-test.out
