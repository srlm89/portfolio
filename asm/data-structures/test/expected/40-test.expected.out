[ 3 2.988470 Basic Instinct 720p ]
[ 3 2.988470 Basic Instinct 720p ][ 4 6.096495 Full.Metal.Jacket.REMASTERED.1987.720p.HDDVD.x264-SEPTiC ]
[ 0 6.216898 007 The Man With the Golden Gun [1974] [BluRay] [720p] [DTS] ][ 3 2.988470 Basic Instinct 720p ][ 4 6.096495 Full.Metal.Jacket.REMASTERED.1987.720p.HDDVD.x264-SEPTiC ][ 14 6.392917 Blood River 720P ]
[ 0 6.216898 007 The Man With the Golden Gun [1974] [BluRay] [720p] [DTS] ][ 3 2.988470 Basic Instinct 720p ][ 4 6.096495 Full.Metal.Jacket.REMASTERED.1987.720p.HDDVD.x264-SEPTiC ][ 14 6.392917 Blood River 720P ][ 32 1.879655 Blade 2   720p ][ 36 1.030676 Pursuit_Of_Happyness[2006]   720p ]
[ 0 6.216898 007 The Man With the Golden Gun [1974] [BluRay] [720p] [DTS] ][ 2 5.955858 Hancock    1080p ][ 3 2.988470 Basic Instinct 720p ][ 4 6.096495 Full.Metal.Jacket.REMASTERED.1987.720p.HDDVD.x264-SEPTiC ][ 14 6.392917 Blood River 720P ][ 26 5.966941 Elite Squad 1080p ][ 32 1.879655 Blade 2   720p ][ 36 1.030676 Pursuit_Of_Happyness[2006]   720p ][ 47 1.412558 Stargate-The Ark of Truth 720p ]
[ 2 5.955858 Hancock    1080p ][ 3 2.988470 Basic Instinct 720p ][ 4 6.096495 Full.Metal.Jacket.REMASTERED.1987.720p.HDDVD.x264-SEPTiC ][ 12 5.559937 Connected   720p ][ 14 6.392917 Blood River 720P ][ 26 5.966941 Elite Squad 1080p ][ 36 1.030676 Pursuit_Of_Happyness[2006]   720p ][ 46 1.135494 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 47 1.412558 Stargate-The Ark of Truth 720p ][ 48 5.298127 X-Men 3    720p ][ 52 5.258948 Crash    1080p ][ 58 7.807691 Fast and the Furious 1 - 1080p ]
[ 2 5.955858 Hancock    1080p ][ 3 2.988470 Basic Instinct 720p ][ 4 6.096495 Full.Metal.Jacket.REMASTERED.1987.720p.HDDVD.x264-SEPTiC ][ 6 1.928617 Minority Report [720p] ][ 11 4.479115 X-Men 1    720p ][ 12 5.559937 Connected   720p ][ 14 6.392917 Blood River 720P ][ 26 5.966941 Elite Squad 1080p ][ 31 1.814512 Hitman    720p ][ 33 6.628069 The.Warlords.2007.720p.BluRay.DTS.x264-ESiR ][ 46 1.135494 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 47 1.412558 Stargate-The Ark of Truth 720p ][ 48 5.298127 X-Men 3    720p ][ 52 5.258948 Crash    1080p ][ 58 7.807691 Fast and the Furious 1 - 1080p ][ 65 6.894192 James Bond 12 - For Your Eyes Only [1981] ]
[ 2 5.955858 Hancock    1080p ][ 3 2.988470 Basic Instinct 720p ][ 4 6.096495 Full.Metal.Jacket.REMASTERED.1987.720p.HDDVD.x264-SEPTiC ][ 11 4.479115 X-Men 1    720p ][ 12 5.559937 Connected   720p ][ 14 6.392917 Blood River 720P ][ 15 9.549769 Pursuit_Of_Happyness[2006]   720p ][ 26 5.966941 Elite Squad 1080p ][ 33 6.628069 The.Warlords.2007.720p.BluRay.DTS.x264-ESiR ][ 42 9.555306 The Secret of Moonarce [720p] ][ 45 0.147915 SAW II   720p ][ 46 1.135494 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 48 5.298127 X-Men 3    720p ][ 52 5.258948 Crash    1080p ][ 58 7.807691 Fast and the Furious 1 - 1080p ][ 60 8.774373 Apollo 13   1080p ][ 62 4.644306 3_10 to Yuma   720p ][ 65 6.894192 James Bond 12 - For Your Eyes Only [1981] ][ 67 2.589482 Alvin And The Chipmunks [720p] ][ 70 8.776040 Final Fantasy The Spirits Within    720p ]
[ 2 5.955858 Hancock    1080p ][ 3 2.988470 Basic Instinct 720p ][ 11 4.479115 X-Men 1    720p ][ 12 5.559937 Connected   720p ][ 14 6.392917 Blood River 720P ][ 15 9.549769 Pursuit_Of_Happyness[2006]   720p ][ 23 7.382940 James Bond 17 - Golden Eye [1995] ][ 33 6.628069 The.Warlords.2007.720p.BluRay.DTS.x264-ESiR ][ 42 9.555306 The Secret of Moonarce [720p] ][ 44 0.140376 Home    1080p ][ 45 0.147915 SAW II   720p ][ 46 1.135494 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 48 5.298127 X-Men 3    720p ][ 51 1.485627 Kung Fu Panda Secrets of the Furious Five    720p ][ 52 5.258948 Crash    1080p ][ 53 2.400935 Finding Nemo [2003] [720p] ][ 58 7.807691 Fast and the Furious 1 - 1080p ][ 60 8.774373 Apollo 13   1080p ][ 62 4.644306 3_10 to Yuma   720p ][ 65 6.894192 James Bond 12 - For Your Eyes Only [1981] ][ 67 2.589482 Alvin And The Chipmunks [720p] ][ 68 9.530507 Disturbia    720p ][ 71 5.903459 X-Men Origin - Wolverine [2009] [1080p] ][ 77 5.391743 Red Cliff    1080p ][ 78 1.748962 Fast and the Furious 2 - 1080p ]
[ 2 5.955858 Hancock    1080p ][ 3 2.988470 Basic Instinct 720p ][ 11 4.479115 X-Men 1    720p ][ 12 5.559937 Connected   720p ][ 15 9.549769 Pursuit_Of_Happyness[2006]   720p ][ 21 3.593027 Defiance.1080p ][ 23 7.382940 James Bond 17 - Golden Eye [1995] ][ 33 6.628069 The.Warlords.2007.720p.BluRay.DTS.x264-ESiR ][ 35 4.908208 Orphan (2009)    1080p ][ 41 2.318450 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p ][ 42 9.555306 The Secret of Moonarce [720p] ][ 45 0.147915 SAW II   720p ][ 46 1.135494 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 48 5.298127 X-Men 3    720p ][ 50 6.673878 Breach 1080p ][ 51 1.485627 Kung Fu Panda Secrets of the Furious Five    720p ][ 52 5.258948 Crash    1080p ][ 53 2.400935 Finding Nemo [2003] [720p] ][ 54 5.593976 Behind Enemy Line 2001   720p ][ 56 9.233027 Oceans.Twelve.2004.720p.HDDVD.x264-CDDHD ][ 59 7.372526 Kingdom of Heaven [2005] [720p] ][ 60 8.774373 Apollo 13   1080p ][ 62 4.644306 3_10 to Yuma   720p ][ 65 6.894192 James Bond 12 - For Your Eyes Only [1981] ][ 67 2.589482 Alvin And The Chipmunks [720p] ][ 71 5.903459 X-Men Origin - Wolverine [2009] [1080p] ][ 77 5.391743 Red Cliff    1080p ][ 82 8.423432 Green Lantern First Flight   720p ][ 85 0.584869 Kung Fu Panda Secrets of the Furious Five    720p ][ 89 4.807683 Hellboy 1    1080p ]
[ 2 5.955858 Hancock    1080p ][ 3 2.988470 Basic Instinct 720p ][ 11 4.479115 X-Men 1    720p ][ 12 5.559937 Connected   720p ][ 15 9.549769 Pursuit_Of_Happyness[2006]   720p ][ 21 3.593027 Defiance.1080p ][ 23 7.382940 James Bond 17 - Golden Eye [1995] ][ 33 6.628069 The.Warlords.2007.720p.BluRay.DTS.x264-ESiR ][ 35 4.908208 Orphan (2009)    1080p ][ 37 8.050939 Darko 720P ][ 42 9.555306 The Secret of Moonarce [720p] ][ 45 0.147915 SAW II   720p ][ 46 1.135494 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 48 5.298127 X-Men 3    720p ][ 50 6.673878 Breach 1080p ][ 51 1.485627 Kung Fu Panda Secrets of the Furious Five    720p ][ 52 5.258948 Crash    1080p ][ 53 2.400935 Finding Nemo [2003] [720p] ][ 54 5.593976 Behind Enemy Line 2001   720p ][ 56 9.233027 Oceans.Twelve.2004.720p.HDDVD.x264-CDDHD ][ 57 1.935369 Face.Off.1997.720p ][ 59 7.372526 Kingdom of Heaven [2005] [720p] ][ 60 8.774373 Apollo 13   1080p ][ 62 4.644306 3_10 to Yuma   720p ][ 64 1.827436 I Am Legend    720p ][ 65 6.894192 James Bond 12 - For Your Eyes Only [1981] ][ 71 5.903459 X-Men Origin - Wolverine [2009] [1080p] ][ 73 0.781529 Gladiator 1080p ][ 82 8.423432 Green Lantern First Flight   720p ][ 83 2.891269 Armageddon [1998] [720p] ][ 84 7.031197 Public Enemies [720p] ][ 85 0.584869 Kung Fu Panda Secrets of the Furious Five    720p ][ 88 3.300553 New York I Love You 720p   720p ][ 89 4.807683 Hellboy 1    1080p ][ 92 7.082095 The Matrix Reloaded   1080p ][ 101 9.016673 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ]
[ 2 5.955858 Hancock    1080p ][ 12 5.559937 Connected   720p ][ 15 9.549769 Pursuit_Of_Happyness[2006]   720p ][ 21 3.593027 Defiance.1080p ][ 23 7.382940 James Bond 17 - Golden Eye [1995] ][ 28 2.529464 Bride Wars   720p ][ 33 6.628069 The.Warlords.2007.720p.BluRay.DTS.x264-ESiR ][ 35 4.908208 Orphan (2009)    1080p ][ 37 8.050939 Darko 720P ][ 42 9.555306 The Secret of Moonarce [720p] ][ 45 0.147915 SAW II   720p ][ 46 1.135494 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 48 5.298127 X-Men 3    720p ][ 50 6.673878 Breach 1080p ][ 51 1.485627 Kung Fu Panda Secrets of the Furious Five    720p ][ 52 5.258948 Crash    1080p ][ 53 2.400935 Finding Nemo [2003] [720p] ][ 54 5.593976 Behind Enemy Line 2001   720p ][ 56 9.233027 Oceans.Twelve.2004.720p.HDDVD.x264-CDDHD ][ 57 1.935369 Face.Off.1997.720p ][ 59 7.372526 Kingdom of Heaven [2005] [720p] ][ 60 8.774373 Apollo 13   1080p ][ 62 4.644306 3_10 to Yuma   720p ][ 64 1.827436 I Am Legend    720p ][ 65 6.894192 James Bond 12 - For Your Eyes Only [1981] ][ 71 5.903459 X-Men Origin - Wolverine [2009] [1080p] ][ 72 4.877273 Hero    1080p ][ 73 0.781529 Gladiator 1080p ][ 74 6.730348 Red Dragon    1080p ][ 80 4.827346 James Bond 20 - Die Another Day [2002] [720p] ][ 83 2.891269 Armageddon [1998] [720p] ][ 88 3.300553 New York I Love You 720p   720p ][ 89 4.807683 Hellboy 1    1080p ][ 92 7.082095 The Matrix Reloaded   1080p ][ 97 3.057022 The Mummy Tomb Of The Dragon Emperor   720p ][ 100 9.400373 Swordfish    720p ][ 101 9.016673 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ][ 102 6.285936 Fantastic Four   1080p ][ 104 7.788449 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 105 8.374442 James Bond 01 - Dr. No [1962] [720p] ][ 108 8.044182 Chicken Run [2000] [720p] ][ 113 8.562154 Around The World In 80 Days [2004] [720p] ]
